src/cosmic_ray/commands/init.py

```diff
@@ -19,32 +25,39 @@ class WorkDBInitVisitor(Visitor):
     for each operator-module pair and running it over the module's AST.
     """
 
-    def __init__(self, module_path, op_name, work_db, operator):
-        self.operator = operator
+    def __init__(self, module_path, op_name, work_db, operator,
+                 interceptors: Interceptors):
```

Comment:
This is not where interceptors belong.

Reply:
Can you explain?

Reply:
The core `init` procedure doesn't need to know anything about interceptors. Interceptors are intentionally a separate, isolated concept. I value this architectural decision more than the potential benefits of the API you've come up with.

Reply:
This architecture extends the possibility of interceptors to intercept.
Init only know Interceptor interface. We always need to know interface, before interceptor interface was `callable(work_db)`.